components/cart/CartItem/CartItem.tsx

```diff
@@ -86,15 +86,17 @@ const CartItem = ({
         <div className="w-16 h-16 bg-violet relative overflow-hidden cursor-pointer z-0">
           <Link href={`/product/${item.path}`}>
             <a>
-              <Image
-                onClick={() => closeSidebarIfPresent()}
-                className={s.productImage}
-                width={150}
-                height={150}
-                src={item.variant.image!.url}
-                alt={item.variant.image!.altText}
-                unoptimized
-              />
+              {item?.variant?.image?.url && (
```

Comment:
Shouldn't this be wrapping the entire link element? Otherwise we have an empty link

Reply:
@goncy I've updated this now to match the behaviour of product cards, where if an image doesn't exist it will show a placeholder image. This way, the link is always available even when there is no image.

<img width="444" alt="Screenshot 2564-12-04 at 17 23 59" src="https://user-images.githubusercontent.com/35846795/144706332-c492b327-77c0-4bda-b514-ddf2a07067b9.png">